securedrop_client/gui/main.py

```diff
@@ -144,10 +153,9 @@ def show_sync(self, updated_on):
         Display a message indicating the data-sync state.
         """
         if updated_on:
-            self.main_view.status.setText('Last refresh: ' +
-                                          updated_on.humanize())
+            self.set_status(_('Last refresh: ' + updated_on.humanize()))
```

Comment:
Also a nit (since we're not doing i18n yet) but strings formatted like this won't work with translations. They need to be:

```python
_('Last fresh: {time} ago').format(updated_on.humanize())
```

Which is to say:
- All substitutions need to be named by keyword
- Only static strings inside the translate functions (`_`, `gettext`, `ngettext`, etc.)